### PR TITLE
Restyle/reformat of the README and UI

### PR DESCRIPTION
**Pull request by khowling**
## PR Summary

* ~~Always refer to the project as "AKS Construction", and the UI (formally "helper") to "AKS Construction initializr"~~
* Updated README to provide a Quick start guide that should be easy to follow.
* Update the helper UI to make more streamlined
* Tidy up GitHub actions deploy UI

New Github Actions Deploy tab (Note the new spacing and labeling)
image

New Readme (Gordon, would like some help here!!)
image

## PR Checklist

- [x] PR has a meaningful title
- [x] Summarized changes
- [ ] This PR is ready to merge and is not **Work in Progress**
- [ ] Link to a filed issue
- [x] Screenshot of UI changes (if PR includes UI changes)

### CODE REVIEW

**Comment by khowling** [comment]
@Gordonby , I cleaned up the Quick Start formatting now, so happy for it to be merged, but please can you check the rest of the README pls.  

I'd love to get this live.

@mosabami are you happy with the rename from helper -> initializr? if not. I can change it back, but still keep the new formatting and README

File: helper/src/configpresets/principals.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "defaultOps": {
-        "title": "Default",
+        "title": "Principles driven",
```
Comment (Gordonby) [inline helper/src/configpresets/principals.json:3]: You mean "Principal driven"

File: README.md [hunk 2]
```diff
@@ -38,28 +53,14 @@ CI Name | Actions Workflow | Parameter file | CI Status | Notes
...
-1. [Connect to your AKS Cluster](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough#connect-to-the-cluster), and deploy your applications as you see fit.
-
-### Mature
+This project unifies guidance provided by the [AKS Secure Baseline](https://docs.microsoft.com/en-us/azure/architecture/reference-architectures/containers/aks/secure-baseline-aks), [Well Architected Framework](https://docs.microsoft.com/en-us/azure/architecture/framework/), [Cloud Adoption Framework](https://azure.microsoft.com/en-gb/cloud-adoption-framework/) and [Enterprise-Scale](https://github.com/Azure/Enterprise-Scale) by providing tangible artifacts to deploy Azure resources from CLI or CI/CD systems.
 
-If you're looking to use this project as part of your deployments, follow these steps.
+The AKS Bicep Accelerator is part of the official [AKS Landing Zone Accelerator (Enterprise Scale)](https://github.com/Azure/AKS-Landing-Zone-Accelerator) architectural approach. To read more about this project and how the AKS Bicep Accelerator fits with Enterprise Scale and the AKS Secure Baseline, look [here](referencearchs.md).
```
Comment (Gordonby) [inline README.md:62]: Aren't we dropping Bicep Accelerator?

**Review by Gordonby — CHANGES_REQUESTED** [review]

**Comment by Gordonby** [comment]
> @Gordonby , I cleaned up the Quick Start formatting now, so happy for it to be merged, but please can you check the rest of the README pls.
> 
> I'd love to get this live.
> 
> @mosabami are you happy with the rename from helper -> initializr? if not. I can change it back, but still keep the new formatting and README

We've pulled the rename to initializr for the time being @mosabami.

**Comment by Gordonby** [comment]
@khowling - we need to also PR this repo to update from Bicep Accelerator.
https://github.com/Azure/AKS-Landing-Zone-Accelerator#bicep-aks-accelerator

**Review by Gordonby — DISMISSED** [review]
LGTM

**Review by Gordonby — APPROVED** [review]
LGTM

**Pull request merged**